molgenis_documentation/user_documentation.md

```diff
@@ -947,6 +951,123 @@ To get the feel of how certain styles look, you can select it in the dropdown me
 
 For an overview of all the different themes offered, visit the [Bootswatch](https://bootswatch.com/) website. It is currently not yet possible to submit your own CSS sheets to update the styling. We do however plan to implement this in the future, giving you even more control to add your own personal style to your MOLGENIS research database.
 
+## <a name="molgenis-modules"></a>Molgenis modules
+### <a name="upload"></a>Upload
+The upload module is the place in Molgenis where you can upload your data into the Molgenis application.
```

Comment:
I think there was decided to use the uppercase spelling for MOLGENIS throughout the document.